setup/enable-backup.sh

```diff
@@ -22,7 +22,7 @@ then
 fi
 
 # Create a bucket
-gsutil mb -l EUROPE-WEST3 -p $project gs://backup-dot-$project
+gsutil mb -l EUROPE-WEST3 -p $project gs://backups-$project
```

Comment:
I don't know. Maybe it was because of the dashboard. Then we remove it here?

Reply:
```suggestion
gsutil mb -l EUROPE-WEST3 -p $project gs://backup-dot-$project
```

Reply:
> I don't know. Maybe it was because of the dashboard. Then we remove it here?

The dashboard uses different buckets and requires special credentials anyway.